experiments/process_sandbox/src/library_runner.cc

```diff
@@ -910,7 +870,7 @@ int main()
   close(PageMapPage);
   callbackSocket.reset(FDSocket);
 
-#ifndef NDEBUG
+  //#ifndef NDEBUG
```

Comment:
Is this commented line still relevant?

Reply:
Ooops, I commented out this block while refactoring and missed readding it.  Thanks!